basics/loops.md

```diff
@@ -470,7 +470,25 @@ while True:
         number = int(number)
     print(numbers)
     ```
-
+5. Make a programme to get a pyramid like shown in the OUTPUT section where user can type the number of rows needed
+   ```
+   OUTPUT for 5 rows
+   1 
+   1 2 
+   1 2 3 
+   1 2 3 4 
+   1 2 3 4 5 
+   ```
+
+6. Make a programme to get a pyramid like shown in the OUTPUT section where user can type the number of rows needed
+   ```
+   OUTPUT when rows is equal to 5
```

Comment:
It would be nice to be consistent with the previous exercise:

```suggestion
   OUTPUT for 5 rows
```

Reply:
Seems like you marked this resolved without actually changing this? Note that "Resolve conversation" and "Commit suggestion" are two different buttons.